wcfsetup/install/files/lib/data/user/online/UsersOnlineList.class.php

```diff
@@ -156,20 +157,24 @@ public function checkRecord() {
 	 * @return	boolean
 	 */
 	public static function isVisible($userID, $canViewOnlineStatus) {
-		if (WCF::getSession()->getPermission('admin.user.canViewInvisible') || $userID == WCF::getUser()->userID) return true;
+		$data = ['result' => false, 'args' => [$userID, $canViewOnlineStatus]];
```

Comment:
Likewise: Please refrain from the args array and opt for explicit array indicies.